plugins/frontend-search/README.md

```diff
@@ -13,54 +12,60 @@ plugins=( ... frontend-search)
 You can use the frontend-search plugin in these two forms:
 
-* `frontend <context> <term> [more terms if you want]`
-* `<context> <term> [more terms if you want]`
+- `frontend <context> <term> [more terms if you want]`
+- `<context> <term> [more terms if you want]`
 
 For example, these two are equivalent:
 
 ```zsh
 $ frontend angularjs dependency injection
+# Will turn into ...
 $ angularjs dependency injection
 ```
 
 Available search contexts are:
 
-| context       | URL                                                                      |
-|---------------|--------------------------------------------------------------------------|
-| angular(>=2.0)| `https://angular.io/?search=`                                             |
-| angularjs(1.x)| `https://google.com/search?as_sitesearch=angularjs.org&as_q=`
-| aurajs        | `http://aurajs.com/api/#stq=`                                            |
-| bem           | `https://google.com/search?as_sitesearch=bem.info&as_q=`                 |
-| bootsnipp     | `https://bootsnipp.com/search?q=`                                         |
-| caniuse       | `https://caniuse.com/#search=`                                            |
-| codepen       | `https://codepen.io/search?q=`                                            |
-| compassdoc    | `http://compass-style.org/search?q=`                                     |
-| cssflow       | `http://www.cssflow.com/search?q=`                                       |
-| dartlang      | `https://api.dartlang.org/apidocs/channels/stable/dartdoc-viewer/dart:`  |
-| emberjs       | `https://emberjs.com/api/#stp=1&stq=`                                     |
-| fontello      | `http://fontello.com/#search=`                                           |
-| html5please   | `http://html5please.com/#`                                               |
-| jquery        | `https://api.jquery.com/?s=`                                             |
-| lodash        | `https://devdocs.io/lodash/index#`                                       |
-| mdn           | `https://developer.mozilla.org/search?q=`                                |
-| npmjs         | `https://www.npmjs.com/search?q=`                                        |
-| qunit         | `https://api.qunitjs.com/?s=`                                            |
-| reactjs       | `https://google.com/search?as_sitesearch=facebook.github.io/react&as_q=` |
-| smacss        | `https://google.com/search?as_sitesearch=smacss.com&as_q=`               |
-| stackoverflow | `https://stackoverflow.com/search?q=`                                     |
-| unheap        | `http://www.unheap.com/?s=`                                              |
+| context       | URL                                                                         |
+| ------------- | --------------------------------------------------------------------------- |
+| angular       | `https://angular.io/docs/ts/latest/api/#!?query=`                           |
```

Comment:
This URL change doesn't work. It redirects to `https://angular.io/api#!?query=` and the query parameter doesn't change the results:

![image](https://user-images.githubusercontent.com/1441704/58038045-30910300-7b2f-11e9-8502-afd3e6f649d0.png)

as opposed to the older URL, which does:

![image](https://user-images.githubusercontent.com/1441704/58038014-1f47f680-7b2f-11e9-8745-2566ef74b630.png)